package.json

```diff
@@ -32,7 +32,7 @@
     "body-parser": "1.15.1",
     "bootstrap": "3.3.6",
     "express": "4.13.4",
-    "ng2-translate": "2.0.0",
+    "ng2-translate": "2.1.0",
```

Comment:
This version of ng2-translate isn't compatible with angular 2.0.0-beta.17, as it uses @angular imports which were only introduced in rc1